src/views/List.jsx

```diff
@@ -14,6 +15,8 @@ export function List({ data, listToken }) {
 		name.toLowerCase().includes(searchField),
 	);
 
+	const sortedList = comparePurchaseUrgency(data);
+
```

Comment:
Instructions for testing this PR:
`comparePurchaseUrgency` accepts data that represents an unsorted list with the items that a user has added, and returns a list sorted according to the acceptance criteria of issue 12. 
To test this create a list (or join an existing one) with items that have a variety of `dateNextPurchased` dates and one or more inactive items (`dateLastPurchased` is over 60 days ago). A sample list you can join is the fifth one in firestore which starts with "co...".